src/app/test/db/mongo/models_test.py

```diff
@@ -557,7 +557,10 @@ def role(self):
 
     @pytest.fixture()
     def role_assignment(self, role):
-        return RoleAssignment(role=role, domain="test_garden")
+        return RoleAssignment(
+            role=role,
+            domain={"scope": "Garden", "identifiers": {"name": "test_garden"}},
+        )
 
     def test_create(self, role_assignment):
```

Comment:
These run fine in the action but would also run locally if there was a mongomock connection established in the `setup_class` method.

Reply:
These tests have always worked locally for me, so I'm not sure what problems you're running into.  Your comment made me go look at the these tests though and they were pretty sloppy in terms of cleaning up after themselves.  I cleaned that up and also added the explicit mongomock connection setup, since the way it was working (through a higher level fixture) is confusing.

Reply:
I'm not sure what's at root either, but I don't think it would be profitable to dig into it too much right now.